server/src/jobs/rdv/anonymizeUsers.ts

```diff
@@ -0,0 +1,27 @@
+import { logger } from "../../common/logger.js"
+import { User } from "../../common/model/index.js"
+
+export const anonimizeUsers = async () => {
+  logger.info("job #anonimizeUsers start")
+  const archiveThreshold = new Date()
```

Comment:
You can use dayjs instead :)

Reply:
it's actually less painful this way ! I did it with dayjs first, but it's more of a hassle because `format()` returns a string, not a date format, and the mongo query expects a Date format...